src/mame/drivers/midvunit.cpp

```diff
@@ -1186,25 +1195,38 @@ PCB Layout
       ADC0844   - National Semiconductor ADC0844 8-Bit Microprocessor Compatible A/D Converter with Multiplexer Option
       PIC16C57  - Microchip PIC16C57
                     - not populated for Cruis'n USA
-                    - labelled 'Offroad 25" U904' for Offroad Challenge
-      PAL1      - GAL20V8 labelled 'A-19668'
+					- labeled 'World Cruis'n 25" U904' for Cruis'n World
+                    - labeled 'Offroad 25" U904' for Off Road Challenge
+      PAL1      - GAL20V8
+                    - labeled 'A-19668' for all three games      
       PAL2      - PALC22V10
-                    - labelled 'A-19669' for Cruis'n USA rev 2.1
-                    - labelled 'A-19993' for Cruis'n USA rev 4.1 (might work with other revs too, but not 2.x)
-                    - labelled 'A-21883 U38' for Offroad Challenge
+                    - labeled 'A-19669' for Cruis'n USA; does not enable linking regardless of version
+                    - labeled 'A-19993' for Cruis'n USA; enables linking. Intended for L4.1. May work in L4.0 but not in any version earlier than that.
```

Comment:
What do you mean "enables linking" or "does not enable linking"?  What are you basing this on?

Reply:
Could've mentioned this earlier http://www.jammarcade.net/cruisin-usa-link-pal-commodore-c16-pls100-shadow-dancer-pal-added/
A-19993's reverse engineer looks pretty clean, but that only helps identify what address 990000 is doing. That is still depending on COMINT coming from U111.

Reply:
Now I wonder if the same process for the two pals is used in Cruis'n World and Off Road Challenge (and please share the link code @Risugami)